examples/connectivity/plot_power_connectome.py

```diff
@@ -0,0 +1,101 @@
+"""
```

Comment:
I could not see the example in CircleCI. Could you rename to new directory? "03_connectivity"

May be ```plot_connectome_power_coords.py```  or ```plot_connectome_power_atlas.py``` ?